src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3792,6 +3792,11 @@ UnboundLambda getUnboundLambda(BoundLambda expr, VariableState variableState)
 
         private void CheckMethodConstraints(SyntaxNode syntax, MethodSymbol method)
         {
+            if (_disableDiagnostics)
```

Comment:
This change seems unrelated to this PR?

Reply:
The changes indirectly triggered duplicate diagnostics in the nullable walker.  This fixed that.

---
In reply to: [301352605](https://github.com/dotnet/roslyn/pull/37052#discussion_r301352605) [](ancestors = 301352605)